dashboard/migrations/0006_remove_all_cache.py

```diff
@@ -0,0 +1,38 @@
+from django.db import migrations, models
+
+
+def forwards_func(apps, schema_editor):
+    """
+    It deletes the cache because it will not be valid after the 0006 migration is applied
+    """
+    CachedEnrollment = apps.get_model("dashboard", "CachedEnrollment")
+    CachedCertificate = apps.get_model("dashboard", "CachedCertificate")
+    CachedCurrentGrade = apps.get_model("dashboard", "CachedCurrentGrade")
+    db_alias = schema_editor.connection.alias
+    CachedEnrollment.objects.using(db_alias).delete()
```

Comment:
From what I understood [here](https://docs.djangoproject.com/en/1.10/ref/migration-operations/#runpython) it is needed to be sure to use the right version of the model.
In this specific case it is probably superfluous.